classes/02class/exercises/c02-network08/alanlima/README.md

```diff
@@ -0,0 +1,48 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    Inbound: it controls incoming traffic TO EC2 instances. Ex.: traffic from local ssh terminal to EC2 instance.
```

Comment:
it's not specific to EC2. it can be for any device in the network. Would you mind adjusting your answer?